elide-core/src/main/java/com/yahoo/elide/core/RequestScopedTransaction.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core;
```

Comment:
We also need to figure out what to do with User in RequestScope.  User cannot be hydrated until a transaction is available.